src/wp-includes/functions.php

```diff
@@ -5249,7 +5249,7 @@ function wp_widgets_add_menu() {
  */
 function wp_ob_end_flush_all() {
 	$levels = ob_get_level();
-	for ( $i = 0; $i < $levels; $i++ ) {
+	for ( $i = 1; $i < $levels; $i++ ) {
```

Comment:
This is a questionable change. PHPUnit marks a test as risky because the output buffer is not closed.

Run the `test_filter_upgrader_pre_install` method with `$i = 0` in `wp_ob_end_flush_all()` to see the problem.

1. `test_filter_upgrader_pre_install()` calls `WP_Upgrader::install_package()` with valid `$source` and `$destination` values.
2. This causes `install_package()` to call `$this->skin->feedback( 'installing_package' );`.
3. This calls `show_message()`, which calls `wp_ob_end_flush_all()`.
4. This results in the following:

```sh
There was 1 risky test:

1) Tests_Admin_IncludesWpUpgrader::test_filter_upgrader_pre_install
Test code or tested code did not (only) close its own output buffers
```